Jucer2Reprojucer/main.cpp

```diff
@@ -543,14 +645,7 @@ int main(int argc, char* argv[])
     wLn();
   }
 
-  auto escapedJucerFileName = jucerFileName.toStdString();
-  std::replace_if(
-    escapedJucerFileName.begin(), escapedJucerFileName.end(),
-    [](const std::string::value_type& c) {
-      return !(std::isalpha(c, std::locale::classic())
-               || std::isdigit(c, std::locale::classic()));
-    },
-    '_');
+  auto escapedJucerFileName = sanitizeName(jucerFileName);
```

Comment:
Please make this cleanup in its own commit. I think there should be 3 commits in this PR:
- one for introducing the `sanitizeName` function,
- one for using the `sanitizeName` function on configuration names, and
- one for using the `sanitizeName` function here.